docs/reference/commandline/plugin_create.md

```diff
@@ -16,9 +16,9 @@ keywords: "plugin, create"
 # plugin create
 
 ```markdown
-Usage:  docker plugin create [OPTIONS] PLUGIN[:tag] PATH-TO-ROOTFS(rootfs + config.json)
+Usage:  docker plugin create [OPTIONS] PLUGIN-DATA-DIR
 
-Create a plugin from a rootfs and configuration
+Create a plugin from a rootfs and configuration. Data dir must contain config.json and rootfs directory.
```

Comment:
"`PLUGIN-DATA-DIR` must contain a configuration file named `config.json` and a root filesystem subdirectory named `rootfs`."